tools/server/server.cpp

```diff
@@ -4076,7 +4077,21 @@ int main(int argc, char ** argv) {
         res_ok(res, {{ "success", true }});
     };
 
-    const auto handle_api_show = [&ctx_server, &res_ok](const httplib::Request &, httplib::Response & res) {
+    const auto handle_api_show = [&ctx_server, &state, &res_ok](const httplib::Request &, httplib::Response & res) {
```

Comment:
If both `api/show` and `api/models` reading the same value from the model, I think we can abstract it out to a new handler, maybe `handle_models_impl` to avoid duplicate the code

Reply:
All values from the model does not seem to be needed, so I removed them and it is back to the previous code.